medicines/doc-index-updater/src/delete_manager/mod.rs

```diff
@@ -42,9 +42,8 @@ impl ProcessRetrievalError for RetrievedMessage<DeleteMessage> {
         e: anyhow::Error,
         state_manager: &StateManager,
     ) -> anyhow::Result<()> {
-        tracing::info!(
-            "Setting error state in state manager for job {}",
-            self.message.job_id
+        tracing::info!(message = "Setting error state in state manager",
```

Comment:
this doesn't look formatted correctly - have you run `cargo fmt` on your changes?

Reply:
nope, will do